loops.js

```diff
@@ -16,9 +18,14 @@
 // loop...
 // loop...
 // klaar!
- // ==========================================
-
-
+// ==========================================
+for (let e = 0; e < 5; e++) {
+    if (e < 4) {
+        console.log('loop...')
+    } if (e >= 4) {
+        console.log('klaar!')
```

Comment:
goed! ik zou alleen "let i"gebruiken omdat het meer conventie is, maar het werkt wel